Makefile

```diff
@@ -3,15 +3,22 @@ SPHINXOPTS    ?= -W --keep-going
 SOURCEDIR     = spec
 BUILDDIR      = _site
 
-.PHONY: default clean build
+.PHONY: default install clean draft spec
 
-default: clean build
+default: install clean spec
```

Comment:
> Is default what runs if you just run `make`? (I thought that was `all`).

I *think* the default is the first defined command.

> Can we also add an `html` target alias for this for those who are used to it from other projects?

Hmm, `make html` is referring to building a singular Sphinx doc usually but we have multiple Sphinx docs, so IMO it's best to avoid possibly going against people's expectations with whatever we would decide `make html` actually do.

Reply:
I don't know if people really think that hard about what `make html` does. They just know that's the incantation to type to build docs.